tests/integration/mutation/create/with_null_input_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package create
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestMutationCreate_WithNullEncrypt_Succeeds(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple create mutation, with null encrypt",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						name: String
+					}
+				`,
+			},
+			testUtils.Request{
+				Request: `mutation {
+					create_Users(encrypt: null, input: {name: "Bob"}) {
```

Comment:
question: `input` is not nillable? (I don't see any tests for that)

Reply:
nice catch! added a test